client/multi-currency-setup/test/index.js

```diff
@@ -0,0 +1,38 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import MultiCurrencySetup from '../tasks/multi-currency-setup';
+
+/**
+ * Internal dependencies
+ */
+import MultiCurrencySetupPage from '..';
+
+jest.mock( '../tasks/multi-currency-setup', () => jest.fn() );
+
+describe( 'MultiCurrencySetupPage()', () => {
+	beforeEach( () => {
+		MultiCurrencySetup.mockReturnValue( <p>Multi-Currency setup task</p> );
+	} );
+
+	afterEach( () => {
+		jest.restoreAllMocks();
+	} );
+
+	it( 'renders "Sell worldwide in multiple currencies" task', () => {
```

Comment:
The naming of this test is a bit strange IMO considering we are moving away from representing this as a task in this PR.  I might be confused here, but it looks like what we are actually testing is whether the setup page renders at all. If so, maybe we change this to:

'renders "Sell worldwide in multiple currencies" page'

Reply:
Agreed and updated. I'm wondering if we should keep this test at all since it's only testing whether the content of the page is rendering, tho. What do you think?

Reply:
Good question. I also was wondering about the need for this test. In the end, I suppose testing that the page renders the expected test is still useful, so I would say to keep it.